Example/Tests/PURLogStoreSpec.swift

```diff
@@ -1,11 +1,12 @@
 import Quick
 import Nimble
+import Puree
 
 class PURLogStoreSpec: QuickSpec {
     override func spec() {
-        describe("readwrite logs", {
+        describe("readwrite logs") {
             let date = NSDate().timeIntervalSince1970
-            let logStoreDBPath = NSTemporaryDirectory().stringByAppendingPathComponent("LogStoreSpec_\(date).db")
+            let logStoreDBPath = NSTemporaryDirectory() + "/LogStoreSpec_\(date).db"
```

Comment:
workaround for following issue
- https://forums.developer.apple.com/thread/13580
- http://qiita.com/melodydance_pr/items/77345499e04de7f214ad